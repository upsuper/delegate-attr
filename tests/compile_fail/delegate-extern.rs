use delegate_attr::delegate;

#[delegate(x)]
#[cfg(not(fake))]
extern "C" {}

fn main() {}
