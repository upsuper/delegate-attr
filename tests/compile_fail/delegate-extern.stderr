error: expected an impl block or method inside impl block
 --> $DIR/delegate-extern.rs:5:1
  |
5 | extern "C" {}
  | ^^^^^^
